src/unified_ai_api.py

```
@@ -46,6 +47,22 @@
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
+# Defensive client compatibility shim for tests: handle closed file objects in httpx multipart
+try:  # pragma: no cover - safety shim only used in tests
+    import httpx  # type: ignore
```

Ruff (0.12.2)

`httpx` imported but unused

Remove unused import: `httpx`

(F401)